### DISCUSSION TITLE
{"error":"unauthorized","reason":"You are not authorized to access this db."}

### DISCUSSION THREAD

**Discussion by saurabhprasadsah** [apache/couchdb#3858] [Q&A]
var id = clientReq.query.docId;
    rest.get('http://localhost:5984/mycompany/' + id,

when you have pass this url you have get response from this url...////

**Comment by nickva** [comment]
I don't know what client library that is. Try to reproduce the issue with curl on the command line and provide few more detailed steps (a script would be ideal).

**Comment by saurabhprasadsah** [comment]
yes when i pass the url..but {"error":"unauthorized","reason":"You are not authorized to access this db."}
